packages/@aws-cdk/aws-kinesisfirehose/lib/lambda-function-processor.ts

```diff
@@ -0,0 +1,32 @@
+import * as lambda from '@aws-cdk/aws-lambda';
+import { Construct } from 'constructs';
+import { DataProcessorBindOptions, DataProcessorConfig, DataProcessorProps, DataProcessorIdentifier, IDataProcessor } from './processor';
+
+/**
+ * Use an AWS Lambda function to transform records.
+ */
+export class LambdaFunctionProcessor implements IDataProcessor {
+  /**
+   * The constructor props of the LambdaFunctionProcessor.
+   */
+  public readonly props: DataProcessorProps;
+
+  private readonly processorIdentifier: DataProcessorIdentifier;
```

Comment:
minor: don't really need this as a class property if it's private and we only use it once